client/src/modules/stock/entry/modals/StockEntryModalForm.service.js

```diff
@@ -28,6 +28,8 @@ function StockEntryModalForm(uuid) {
     this.isInvalid = true;
     this.isValid = false;
     this.identifier = uuid();
+    this.number_box = row.number_box || null;
+    this.package_size = row.package_size || null;
```

Comment:
It seems like the this uses different terms:  package vs box.  I suggest using one for both.  Eg:  this.number_packages and this.package_size. (or with "box").  "Package" may be a little better than "box" since "package" seems more generic.  I can imagine that meds might come in tubes or barrels containing multiple doses.  In that case "box" seems awkward.

Reply:
I will do it